matrix/instrumentations.yaml

```diff
@@ -0,0 +1,340 @@
+- keys:
+    - ASPNET
+  instrumented_libraries:
+    - name: ASP.NET Framework (.NET Framework)
+      supported_versions: See :ref:`dotnet-otel-versions`
+  comment: MVC / WebApi (Only integrated pipeline mode supported). Metrics requires trace instrumentation.
+  stability: beta
+  supportability_level: community
+  dependencies:
+    - name: ASP.NET Instrumentation for OpenTelemetry
+      link: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.AspNet
+      version: 1.0.0-rc9.9
+      stability: beta
+    - name: ASP.NET Telemetry HttpModule for OpenTelemetry
+      link: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule
+      version: 1.0.0-rc9.9
+      stability: beta
+  to_do_comment: supportedVersions with ref link
+  signals:
+    - traces: 
+    - metrics:
+        - metric: http.server.duration_{bucket|count|sum}
```

Comment:
Are there no attributes for these metrics?

Reply:
Not documented. https://docs.splunk.com/observability/en/gdi/get-data-in/application/otel-dotnet/configuration/dotnet-metrics-attributes.html

We can improve this, when current scenario will be covered.